pkgs/development/python-modules/enochecker_core/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "enochecker_core";
+  version = "0.10.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-N41p2XRCp55rcPXLpA4rPIARsva/dQzK8qafjzXtavI=";
+  };
+
```

Comment:
Set `doCheck = false` with a comment that there is no test upstream.

Reply:
and please add `pythonImportsCheck`.

Reply:
done. Thanks for hinting at `pythonImportsCheck` :)